tests/js/client/aql/aql-query-cache-noncluster.js

```diff
@@ -62,13 +46,7 @@ function ahuacatlQueryCacheTestSuite () {
 ////////////////////////////////////////////////////////////////////////////////
 
     setUp : function () {
-      let command = `
-        let cacheProperties = AQL_QUERY_CACHE_PROPERTIES();
-        AQL_QUERY_CACHE_INVALIDATE();
-        return cacheProperties;
-      `;
-      cacheProperties = arango.POST("/_admin/execute", command);
-
+      cacheProperties = cache.properties({});
```

Comment:
```suggestion
      cacheProperties = cache.properties();
```

Reply:
The previous code also called `AQL_QUERY_CACHE_INVALIDATE()`, i.e. it cleared the cache.
Should we also clear the cache here via an additional `cache.clear();`?